CalibCalorimetry/HcalAlgos/interface/HcalHardcodeParameters.h

```diff
@@ -38,6 +39,7 @@ class HcalHardcodeParameters {
 		//member variables
 		double pedestal_, pedestalWidth_;
 		std::vector<double> gain_, gainWidth_;
+		int zsThreshold_;

```

Comment:
@kpedro88 sorry for pointing you only now to it, but wouldn't it be better to make the members all const, as you do not change them after constructing the object?

Reply:
Unfortunately this does not work, because then instances of this class can't be copy-assigned (which is done in `HcalDbHardcode`). However, I noticed a bunch of accessors in `HcalDbHardcode` could be made const, so I'll do that.

Reply:
ok, thanks for clarifying!